androidHackathonApp/src/main/res/values/dimens.xml

```diff
@@ -1,12 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    //Margins
```

Comment:
I didn't realise you could do `//` in the xml, I thought it had to be the old `<!--` way

Reply:
I'm not actually sure if `<!--` is still standard or not, to be honest. But I've used `//` for ages and no one's said anything lol

Reply:
Much prefer //!